db/seeds.rb

```diff
@@ -9,4 +9,10 @@
 employee_2 = Employee.create(email: "dwojka@gmail.com", password: "222222")
 employee_3 = Employee.create(email: "trojka@gmail.com", password: "333333")
 employee_4 = Employee.create(email: "czworka@gmail.com", password: "444444")
-employee_5 = Employee.create(email: "piatka@gmail.com", password: "555555")
\ No newline at end of file
+employee_5 = Employee.create(email: "piatka@gmail.com", password: "555555")
+
+kudo1 = Kudo.create! :title => Faker::Beer.style, :content => Faker::Coffee.notes, :giver_id => 1, :receiver_id => 5
+kudo2 = Kudo.create! :title => Faker::Beer.style, :content => Faker::Coffee.notes, :giver_id => 2, :receiver_id => 4
+kudo3 = Kudo.create! :title => Faker::Beer.style, :content => Faker::Coffee.notes, :giver_id => 3, :receiver_id => 3
+kudo4 = Kudo.create! :title => Faker::Beer.style, :content => Faker::Coffee.notes, :giver_id => 4, :receiver_id => 2
+kudo5 = Kudo.create! :title => Faker::Beer.style, :content => Faker::Coffee.notes, :giver_id => 5, :receiver_id => 1
```

Comment:
This is a lot of repetition, maybe you can create all those kudos with a loop.

Also, there are a lot of useless assignements - you are assigning newly created objects to a variable (like kudo1) which is never used again.

Bonus question - why are you creating employees, with `create`, but kudos with `create!`

Also why you using two different types of passing arguments to create/create methods - pick one, so it is simpler to read ;)

Reply:
I changed adding kudos to the loop.
I wanted to see if there would be any difference in this case create/create!
I wanted to try to use creation in a different way that I found. But this method => is probably not used anymore because it's an old syntax